### ISSUE TITLE
[Accessibility][Bypass Blocks - Design Flow]: Ensures each page has at least one mechanism for a user to bypass navigation and jump straight to the content at property pane.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2538]
**User Experience** 
Skip to main content provides a way to skip the repeated content, due to their absence user will have to navigate through the same content again and again whenever they refresh or reload the website.

**Test Environment**
Browser: Microsoft Edge Dev Version 83.0.467.0 (Official build)
OS build: 2004 (19588.1000)
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. "Bot Framework Composer" homepage will gets opened.
3. Select any of the previously created bot from "Recent Bots" or create one from the available Bots under 'Examples' and activate it.
4. Run Accessibility insight and observe the issue.

**Actual:** 
The page doesn't have at least one mechanism for a user to bypass navigation and jump straight to the content.

**Expected:**
Ensures each page has at least one mechanism for a user to bypass navigation and jump straight to the content.

**Element path:** 
html

**Snippet:** 
<html lang="en" dir="ltr">

**How to fix:** 
Fix any of the following:
  No valid skip link found
  Page does not have a header
  Page does not have a landmark region
  No valid skip link found
  Page does not have a header
  Page does not have a landmark region

**Remark:**
This accessibility issue was found using Accessibility Insights for Web 2.16.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Note**
Same issue observed throughout the website for every property pane as well as the flow design.

**MAS Impacted:** [MAS2.4.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={20bce2aa-9952-4a10-af79-3376bcc8766d})

**Attachment:**
![MAS2 4 1_Skip to main content is not available](https://user-images.githubusercontent.com/60913131/78645113-95cf1d80-78d4-11ea-84bf-0e897ac006d9.png)

**Comment by yeze322** [comment]
URL router @lei9444

**Comment by beyackle** [comment]
@ashish315 Could I get a re-check on this? The Insights tool isn't complaining about the above any more.

**Comment by ashish315** [comment]
@beyackle Agree with you. So closing the issue.
